locale/de_DE.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -164,24 +164,32 @@ msgid ""
 "With PalMA, you can share your documents and your desktop\n"
 "            with your learning group."
 msgstr ""
+"Mit PalMA können Sie Ihre Dokumente und Ihren Desktop\n"
+"            mit Ihrer Lerngruppe teilen."
 
 #: index.php:1113
 msgid ""
 "Team members can join the group at any time. All they need\n"
 "            is URL and PIN."
 msgstr ""
+"Teammitglieder können der Gruppe jederzeit beitreten. Sie\n"
+"            brauchen dazu nur die URL und die PIN."
 
 #: index.php:1117
 msgid ""
 "The PalMA team monitor shows up to 4 contributions\n"
 "            simultaneously."
 msgstr ""
+"Der PalMA-Team-Monitor zeigt bis zu 4 Beiträge\n"
+"            gleichzeitig an."
 
 #: index.php:1119
 msgid ""
 "Just upload PDF files, images, OpenDocument or\n"
 "            MS Office files or enter a URL &ndash; PalMA will show them."
 msgstr ""
+"PDF-Dateien, Bilder, Open-Office- oder MS-Office-Dateien"
```

Comment:
We need a line break here (otherwise the line becomes too long).
I also suggest to keep OpenDocument in the German translation.